index.test.js

```diff
@@ -613,9 +613,36 @@ describe('Real-world usage', () => {
         .doesntMatter(Equivalency.WHITESPACE_DIFFERENCES)
         .doesntMatter(Equivalency.CAPITALIZATION)
         .doesntMatter(Equivalency.en.COMMON_PUNCTUATION)
+        .matters('-')
         .doesntMatter(Equivalency.en.COMMON_SYMBOLS)
         .doesntMatter(Equivalency.COMMON_DIACRITICS)
-        .matters('-');
+        .doesntMatter(Equivalency.HYPHENS_OMITTED_OR_REPLACED_WITH_SPACES);
+
+      it('should handle hyphens correctly', () => {
+        const target = 'over-the-moon cow';
```

Comment:
A term we might want to start using for this concept -- the "fully correct" answer -- is "canonical". It's used in SEO: a canonical url is that one that you want google to index, like www.example.com/route, and you write your server logic such that any other url that serves a page that is the same or very similar has a `<link rel="canonical" href="www.spanishdict.com/list/1/slug">` tag in it, such as:
- www.spanishdict.com/list/1/something-other-than-slug
- www.spanishdict.com/list/1/slug?query=value

Then google counts any links towards those urls as links towards the canonical url, which helps you aggregate search rank on the canonical instead of it being distributed among several (possibly many, depending on what you do with urls) urls. Anyway, I think for us we've got a canonical answer which is what's in the db, and then a space of 1 or more correct answers for that canonical answer -- which strings are in that space is determined by the rules of the equivalency in use. Using this terminology would help to distinguish between the answer that is in the db and a user input that is not identical to that answer but will marked as correct.

Reply:
I like that idea. I'm going to leave `target` here for now while we figure out the details of unidirectional rules.